src/styles/tailwind.css

```diff
@@ -159,14 +163,6 @@ If your plugin does not need CSS, delete this file.
   opacity: 1;
 }
 
-.context-note .note-name {
```

Comment:
These 3 red blocks were added recently, why are they removed here?

Reply:
@logancyang Maybe I don't use the latest `styles.css` in master branch. I will replace it by  the lasted `styles.css`